### DISCUSSION TITLE
[help] Custom S3 Storage

### DISCUSSION THREAD

**Discussion by rsangole** [ropensci/targets#1246] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hello,

Is is possible to use a custom (internally hosted) AWS-like S3 bucket to store the targets cache?

- Rahul

**Comment by noamross** [comment, marked as answer]
Yup! Any service with a S3-compatible API will do.  I have an example set up here using a local [MinIO](https://min.io/) instance on a single machine: https://github.com/noamross/targets-minio-versioning.  I have tested https://wasabi.com/ as well and it worked fine.

**Reply by rsangole** [reply]
Thanks @noamross . I'll read through this.
